notebooks/500__jupyterhub/elyra/run-pipelines-on-data-science-pipelines/hello-generic-world.pipeline

```diff
@@ -14,16 +14,22 @@
           "op": "execute-notebook-node",
           "app_data": {
             "component_parameters": {
-              "filename": "load_data.ipynb",
-              "runtime_image": "docker.io/amancevice/pandas:1.1.1",
+              "dependencies": [],
+              "include_subdirectories": false,
               "outputs": [
                 "data/noaa-weather-data-jfk-airport/jfk_weather.csv"
               ],
               "env_vars": [
                 "DATASET_URL=https://dax-cdn.cdn.appdomain.cloud/dax-noaa-weather-data-jfk-airport/1.1.4/noaa-weather-data-jfk-airport.tar.gz"
               ],
-              "dependencies": [],
-              "include_subdirectories": false
+              "kubernetes_pod_annotations": [],
+              "kubernetes_pod_labels": [],
+              "kubernetes_secrets": [],
+              "kubernetes_shared_mem_size": {},
+              "kubernetes_tolerations": [],
+              "mounted_volumes": [],
+              "filename": "load_data.ipynb",
+              "runtime_image": "quay.io/modh/runtime-images@sha256:27f12a510a034212ce4d579a970cce7aeeb33ffa32044fe88a262ae15d34e763"
```

Comment:
I don't think these changes are needed, especially the fields added by what looks like our Jupyterlab instance
```
 "kubernetes_pod_annotations": [],
              "kubernetes_pod_labels": [],
              "kubernetes_secrets": [],
              "kubernetes_shared_mem_size": {},
              "kubernetes_tolerations": [],
              "mounted_volumes": [],
```